### PR TITLE
Test samples for pending PR.

### PR DESCRIPTION
**Pull request by nagmat84**
This PR only adds the test samples to the `master` branch which are required by #1351. The test cases for import via URL download use URLs like these

```php
public const SAMPLE_DOWNLOAD_JPG = 'https://github.com/LycheeOrg/Lychee/raw/master/tests/Samples/mongolia.jpeg';
public const SAMPLE_DOWNLOAD_TIFF = 'https://github.com/LycheeOrg/Lychee/raw/use_filestreams/tests/Samples/tiff.tif';
```

To ensure that the test suite runs for the development branch `use_filestreams` before the merge and the `master` branch after the merge all URLs must already point to the master branch. However, this requires that the sample files are independently merged into master at first.

I added the EXIF tag `Location` to the header of the sample file `night.jpg`, because the related branch in the EXIF reader was untested. That's why the file is 266 bytes larger and appears as changed.

### CODE REVIEW

File: tests/TestCase.php [hunk 1]
```diff
@@ -20,20 +20,59 @@ abstract class TestCase extends BaseTestCase
...
+	public const MIME_TYPE_IMG_JPEG = 'image/jpeg';
+	public const MIME_TYPE_IMG_PNG = 'image/png';
+	public const MIME_TYPE_IMG_TIFF = 'image/tiff';
+	public const MIME_TYPE_IMG_WEBP = 'image/webp';
+	public const MIME_TYPE_IMG_XCF = 'image/x-xcf';
+	public const MIME_TYPE_VID_MP4 = 'video/mp4';
+	public const MIME_TYPE_VID_QUICKTIME = 'video/quicktime';
+
+	public const SAMPLE_DOWNLOAD_JPG = 'https://github.com/LycheeOrg/Lychee/raw/master/tests/Samples/mongolia.jpeg';
+	public const SAMPLE_DOWNLOAD_TIFF = 'https://github.com/LycheeOrg/Lychee/raw/use_filestreams/tests/Samples/tiff.tif';
```
Comment (qwerty287) [inline tests/TestCase.php]: I'm not sure if I like this. I often develope offline which will break these, also we don't need the additional traffic if we have the files locally.

Comment (nagmat84) [reply tests/TestCase.php]: We need to download them to test the feature "import via URL". This is something completely different than import from a local file and also takes a different code path in Lychee. For example, the check whether the MIME type of a file is acceptable differs depending on whether the file is locally available are needs to be downloaded. I already found some bugs in the "import via URL code" this way. So we need both: local and download.

Comment (qwerty287) [reply tests/TestCase.php]: Ah okay, then I am completely wrong here. Sorry.

Comment (nagmat84) [reply tests/TestCase.php]: No problem. Questions are always welcome. Most tests are actually run locally as you can see from the path constants some lines below. That's also the reason why there are only two files to be tested with a proper download: a JPEG and a TIFF.

TIFF is treated as "raw" by Lychee (and thus has a different code path than JPEG) but is still supported my Imagick. Everything else which can be tested about images (EXIF, auto-rotation, broken binary encoding, etc.) is tested with import from local files.

**Review by ildyria — APPROVED** [review]

**Pull request merged**